CONTRIBUTING.md

```diff
@@ -11,6 +13,26 @@
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
+## Commit sign-off
+
+To commit your changes, you would need to sign off on your commit:
+
+```
+git commit -s -m ....
+```
+
+To amend a previous commit:
+
+```
+git commit --amend --signoff
+```
+
+You can also mass sign-off a whole pull request with
+
+```
+git rebase --signoff ${YOUR_BRANCH_NAME}
```

Comment:
Same comment here as on the other PR:
I think YOUR_BRANCH_NAME should probably say BASE_BRANCH_NAME, otherwise it might sound like you should give the name of your current branch, which does not work.

Reply:
Thanks! Will update it

Reply:
Updated :)